src/utils.ts

```diff
@@ -168,3 +168,14 @@ export async function zoomTo(viewer: Viewer, config): Promise<void> {
     viewer.zoomTo(p);
   }
 }
+
+export function debounce(f, ms) {
+  let isCooldown = false;
+  return (...args) => {
+    if (isCooldown) return;
+    f(...args);
+    isCooldown = true;
+    setTimeout(() => isCooldown = false, ms);
```

Comment:
Nice to have: ideally the function would also be called at the end of the timeout because currently, if a call is triggered 250ms before the mouse hovers the button the cursor will not change styling.